plugins/jobs/src/js/components/form/helpers/JobFormData.ts

```diff
@@ -152,7 +153,7 @@ export interface JobPlacement {
 }
 
 export enum ConstraintOperator {
-  Eq = "EQ",
+  Is = "IS",
```

Comment:
why is "EQ" not in here?

Reply:
`IS` and `EQ` are the same - feedback from product/metronome team was to choose one to display in form (we use `IS` because that's what is used in services). We include `EQ` in validation because it is supported if someone adds a constraint with `EQ` to JSON

Reply:
ah, thanks! 🙇